Src/Plugins/Security/Utilities/CertificateHelper.cs

```diff
@@ -73,6 +73,11 @@ internal static class CertificateHelper
                                                          ref Fingerprint fingerprint,
                                                          ref string message)
         {
+            // @michaelcfanning 3/28/23
```

Comment:
>michaelcfanning [](http://example.com/codeflow?start=16&length=15)

an example of our TODO format, user name, date, link to GH issues, good description of open item. please absorb and model this standard. :)